src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  posts: Post[]
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo
+        key={post.id}
+        post={post}
+      />
```

Comment:
no need to move attributes to a new line if you have only two of them and they perfectly fit into a single line